mobile/library/common/config/config.cc

```diff
@@ -538,5 +539,27 @@ stats_sinks: *stats_sinks
 R"(
         overload:
           global_downstream_max_connections: 0xffffffff # uint32 max
+#{custom_layers}
 )";
+
+const char* rtds_layer_insert = R"(
+    - name: {}
+      rtds_layer:
+        name: {}
+        rtds_config:
+          initial_fetch_timeout:
+            seconds: 1
```

Comment:
A 1 second timeout seems very aggressive considering real world mobile networking conditions.

I'm not familiar with the implications of this timeout though.

Reply:
agree, this should probably be 5s to start off with.  we can experiment with it as necessary.

Reply:
I'll make it configurable but I think it needs to be 1 second in tests because the tests depend on the initial fetch timing out

Reply:
Why not expose a builder function to configure this then?

Reply:
Just did! Sorry I wasn't clear haha

Reply:
Great, sorry I missed that.